simulators/eth2/testnet/main.go

```diff
@@ -1,18 +1,13 @@
 package main
 
 import (
-	"context"
-	"encoding/json"
 	"github.com/ethereum/hive/hivesim"
-	"time"
+	"github.com/ethereum/hive/simulators/eth2/testnet/setup"
 )
 
-func jsonStr(v interface{}) string {
-	dat, err := json.MarshalIndent(v, "  ", "  ")
-	if err != nil {
-		panic(err)
-	}
-	return string(dat)
+var tests = []testSpec{
+	{Name: "transition-testnet-basic-geth-lighthouse", Fn: Bscmtt{node{"go-ethereum", "lighthouse"}}},
+	{Name: "transition-testnet-basic-geth-teku", Fn: Bscmtt{node{"go-ethereum", "teku"}}},
```

Comment:
I would suggest we use the provided clients to generate the client combinations instead of hard-coding the combinations here.

E.g. if we have:
Execution Clients: A
Beacon Client: B, C, D
Validator Client: E, F

The resulting testnet could be comprised of client combos:
A/B/E + A/C/E + A/D/E + A/B/F + A/C/F + A/D/F

Reply:
Done, thanks.